static/js/faculty_carousel.js

```diff
@@ -5,7 +5,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 const carouselDiv = document.querySelector('#faculty-carousel');
-ReactDOM.render(
-  <FacultyCarousel faculty={SETTINGS.faculty} />,
-  carouselDiv
-);
+if (carouselDiv !== null) {
+  ReactDOM.render(
+    <FacultyCarousel faculty={SETTINGS.faculty}/>,
+    carouselDiv
+  );
+}
```

Comment:
OK so looking at `cms/templates/cms/program_page.html` we'll have `#faculty-carousel` if there are faculty to show, and otherwise we won't.

I was worried that the error was because of a race condition, but I see that's not a concern.